alphagov/paas-cf
# [#149566861] Remove CI deployment part1

## What

This is the first part of removing the CI deployment. This updates staging to match the current CI deployment, and also makes it trigger from master instead of from the tags that CI creates.

Once this is deployed, there will be a second PR to remove the CI deployment etc.

## How to review

Code review:
* Verify the changes make sense
* Verify this brings staging into line with CI
* Verify these changes only affect staging

## Who can review

Not me.
